FordLabs/peoplemover
# Move read only flag from redux to recoil

## Issue
Resolves N/A

## What was done
- [x] Move read only flag from redux to recoil

## How to test
1. Ensure everything works as it did before.
2. Manually switch this flag to true (https://github.com/FordLabs/peoplemover/blob/move-is-read-only-to-recoil/ui/src/Auth/AuthorizedRoute.tsx#L39) and ensure that everything in a space is now read only
